README.md

```diff
@@ -60,6 +60,9 @@ Alternatively, you can use docker-compose:
 version: "3.8"
 services:
   pacoloco:
+#   if a specific user id is provided, you have to make sure
+#   the mounted directories have the same user id owner on host
```

Comment:
Since the docker daemon is root, could we `chown -R` the directory to the given `uid:gid` on container start(?). I wonder whether that's desirable or less secure?

Reply:
I think this could be done with the init container specifically (start up the `busybox` container with mounted volumes, wait for it to finish and only then bring up the pacoloco itself).
I'm not sure about the security implications tbh. I guess that if you provide a specific UID:GID pair it's reasonable to expect all the files in the mounted volumes to be readable/writable by that user?

Anyhow, it'd look roughly like this:
```
  fixup-permissions:
    image: busybox
    volumes:
      - ...
    command: chown -R UID:GID /the_mount_point
pacoloco:
   ...
    depends_on:
        fixup-permissions:
            condition: service_completed_successfully
```
I can describe it in the README if you think it makes sense.

Reply:
This might be overdoing it, then.